tools/workshop/generate-profiles-from-seeds.ps1

```
@@ -0,0 +1,140 @@
+param(
+    [Parameter(Mandatory = $true)][string]$SeedPath,
+    [string]$OutputRoot = "profiles/custom",
+    [string]$NamespaceRoot = "custom",
```

PSScriptAnalyzer (1.24.0)

[warning] The parameter 'NamespaceRoot' has been declared but not used.

(PSReviewUnusedParameter)